src/main-dashboard.js

```diff
@@ -38,7 +40,12 @@ document.addEventListener('DOMContentLoaded', function() {
 	const register = OCA?.Dashboard?.register || (() => {})
 
 	register('mail', (el) => {
```

Comment:
maybe rename this one to `mail-important` then?

Reply:
I deliberately chose not to because it breaks existing dashboards. Users would have to manually add the important mail widget again.

Reply:
Ah yes. Good point :+1: